ultravox/model/ultravox_model.py

```diff
@@ -162,8 +206,25 @@ def forward(
             past_key_values=past_key_values,
             **kwargs,
         )
-
-        return lm_output
+        if self.training:
```

Comment:
Hmm, why gate this with `self.training`?
Do you explicitly want the loss to be different in training vs validation?

Is is because you want the validation loss to be comparable to other runs?